### DISCUSSION TITLE
[feature_toggle] Collapse TopNav header by default

### DISCUSSION THREAD

**Discussion by Foorack** [grafana/grafana#67573] [Dashboards, Panels & Visualizations]
Would it please be possible to add a feature toggle for collapsing the header by default? We use Grafana for the metrics page of a game, where users can anonymously see the graphs, and the TopNav with "search" and "Log in" in this case is only distracting, as we don't want users to do either of those things. There is only 1 Dashboard.

![image](https://user-images.githubusercontent.com/5008081/235298592-64152467-9e47-4681-987c-cc7a2ff57cec.png)

**Comment by baur** [comment]
same problem https://github.com/grafana/grafana/discussions/67650

You can open with param &kiosk=tv
but I don't know is this a feature or a bug )))

**Comment by ashharrison90** [comment]
@Foorack we don't plan on adding a config parameter for that behaviour in the immediate term.

i believe we have a mechanism in place currently to accomplish what you want. append `?kiosk=true` or just `?kiosk` to your url.

this mode will hide both navigation bars and disable the command palette (press escape to leave this mode) 👍

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This question has been answered, so I will be closing this discussion. Feel free to make an [issue](https://github.com/grafana/grafana/issues) if the answer is not sufficient and you would like to request the feature.
